src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
```

Comment:
use `this` at your methods
```suggestion
    add(num) {
       this.result += num;
    },
```

Reply:
use this.result += num;